pkg/host/analyzer.go

```diff
@@ -112,16 +112,14 @@ func (a *analyzer) start() {
 	}()
 }
 
-//shouldInsertNewRecord returns true if a host entry with the current CID does not exist in the database
-func (a *analyzer) shouldInsertNewHostRecord(ssn *mgo.Session, host data.UniqueIP) bool {
-	var hostCIDs []struct {
-		CID int `bson:"cid"`
-	}
+//shouldInsertNewHostSubdocument returns true if a host entry with the current CID does not exist in the database
+func (a *analyzer) shouldInsertNewHostSubdocument(ssn *mgo.Session, host data.UniqueIP) bool {
```

Comment:
renamed because I was confused as I thought this function was checking whether a new top level entry should be inserted into the host collection